Library/Formula/shc.rb

```diff
@@ -0,0 +1,23 @@
+class Shc < Formula
+  desc "Shell Script Compiler"
+  homepage "https://neurobin.github.io/shc"
+  url "https://github.com/neurobin/shc/archive/3.9.3a.tar.gz"
```

Comment:
It stands for alpha, but actually the only thing that changed from 3.9.2 is the PR I submitted to fix the install script permissions. I suppose the developer didn't want to “waste” a full version on this small fix.